includes/class-wc-stripe-helper.php

```diff
@@ -1983,4 +1983,24 @@ public static function validate_intent_for_order( $order, $intent, ?string $sele
 
 		throw new Exception( __( "We're not able to process this request. Please try again later.", 'woocommerce-gateway-stripe' ) );
 	}
+
+	/**
+	 * Determines if the store is connected to Stripe.
+	 *
+	 * @param string $mode Optional. The mode to check. 'live' or 'test' - if not provided, the currently enabled mode will be checked.
+	 * @return bool True if connected, false otherwise.
+	 */
+	public static function is_connected( $mode = null ) {
```

Comment:
Non-blocking nit: for new code like this, does it make sense to add explicit types?
```suggestion
	public static function is_connected( ?string $mode = null ) {
```
On second glance, we're calling the function directly from the existing `is_connected()` function, which doesn't have strict types, so maybe it's not worth adding that for now.